metamod/Makefile

```diff
@@ -169,10 +169,10 @@ endif
 # "WONT WORK WITH omit-frame-pointer"?
 # - disable (unneeded) C++ exceptions and rtti code to save some space ?
 
-CCOPT = $(CCO) $(CCOPT_ARCH) -fno-exceptions -fno-rtti
+CCOPT = $(CCO) $(CCOPT_ARCH) -fno-exceptions -fno-rtti -ftree-vectorize
```

Comment:
I did manual inspection of all new generated SSE2 instructions generated (not many, 95 instructions for win32 dll and 61 for linux dll) with this option and I do not see improvement. The changes in code generation are more likely to cause slowdown than speed up (unmeasurable as SSE2 instruction count is so low compared to the rest).

The main reason for enabling SSE3 was its new x87 instruction FISTTP which speeds up floating-point to integer conversions.